test/test.js

```diff
@@ -15,66 +15,73 @@ function doesNotInclude(string, substring) {
 }
 
 function makeBundle(entries) {
-  return rollup({ entry: entries, plugins: [multiEntry()] });
+  return rollup({ input: entries, plugins: [multiEntry()] });
 }
 
 describe('rollup-plugin-multi-entry', () => {
-  it('takes a single file as input', () =>
-    makeBundle('test/fixtures/0.js').then(bundle => {
-      const code = bundle.generate({ format: 'cjs' }).code;
-      includes(code, 'exports.zero = zero;');
+  it('takes a single file as input', () => {
+    return makeBundle('test/fixtures/0.js').then(bundle => {
+      return bundle.generate({ format: 'cjs' }).then(({code}) => {
+        includes(code, 'exports.zero = zero;');
+      });
     })
-  );
+  });
 
-  it('takes an array of files as input', () =>
-    makeBundle(['test/fixtures/0.js', 'test/fixtures/1.js']).then(bundle => {
-      const code = bundle.generate({ format: 'cjs' }).code;
-      includes(code, 'exports.zero = zero;');
-      includes(code, 'exports.one = one;');
+  it('takes an array of files as input', () => {
+    return makeBundle(['test/fixtures/0.js', 'test/fixtures/1.js']).then(bundle => {
+      return bundle.generate({ format: 'cjs' }).then(({code}) => {
+        includes(code, 'exports.zero = zero;');
+        includes(code, 'exports.one = one;');
+      });
     })
-  );
+  });
 
-  it('allows an empty array as input', () =>
-    makeBundle([]).then(bundle => {
-      const code = bundle.generate({ format: 'cjs' }).code;
-      doesNotInclude(code, 'exports');
+  it('allows an empty array as input', () => {
+    return makeBundle([]).then(bundle => {
+      return bundle.generate({ format: 'cjs' }).then(({code}) => {
+        doesNotInclude(code, 'exports');
+      });
     })
-  );
+  });
 
-  it('takes a glob as input', () =>
-    makeBundle('test/fixtures/{0,1}.js').then(bundle => {
-      const code = bundle.generate({ format: 'cjs' }).code;
-      includes(code, 'exports.zero = zero;');
-      includes(code, 'exports.one = one;');
+  it('takes a glob as input', () => {
+    return makeBundle('test/fixtures/{0,1}.js').then(bundle => {
+      return bundle.generate({ format: 'cjs' }).then(({code}) => {
```

Comment:
Sorry for delay. Please split promise hell in chain please like
```
makeBundle()
  .then(bunde => bundle.generate())
  .then(({ code }) => {
    // process
  })
```
Also please don't change code style of it curly brackets.

Reply:
I didn’t think it worth holding out for this change.